main.py

```diff
@@ -1,10 +1,39 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        player_data = json.load(file)
+
+    for player, value in player_data.items():
+        race = Race.objects.get_or_create(
+            name=value["race"]["name"],
+            description=value["race"]["description"]
+        )[0] if value["race"] else None
```

Comment:
Using if else assignment here leads to additional complexity and degrades the code readability. Use another approach for `Race` and `Guild` 
```suggestion
        race = None
        if race_value := value.get("race"):
            race, _ = Race.objects.get_or_create(
                name=race_value["name"],
                description=race_value["description"]
            )
```